### PR TITLE
feat: 🎸 Pie and donut value mode

### PR DESCRIPTION
**Pull request by dariuszlacheta**

### CODE REVIEW

File: packages/charts/src/charts/donut/donut-chart.component.tsx [hunk 1]
```diff
@@ -20,7 +20,12 @@ import DonutTotal from './donut-total.component';
 
 import { theme as defaultTheme } from '../../theme';
 
-import { CommonChartSettings, ItemData, TooltipSettings } from '../../types';
+import {
+  CommonChartSettings,
+  ItemData,
+  TooltipSettings,
+  CircularValueMode,
```
Comment (maciejrybaniec) [inline packages/charts/src/charts/donut/donut-chart.component.tsx:27]: `CircularValueMode` - suggests the selected mode is `Circular`. I'd rather suggest to rename it to something like `CircularChartValueMode`

File: packages/charts/src/charts/pie/pie-slice.component.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ type Props = {
   autocolor: boolean;
   labelPosition: [number, number];
   activePosition: [number, number];
-  label: string;
+  label: string | number | Date | boolean;
```
Comment (maciejrybaniec) [inline packages/charts/src/charts/pie/pie-slice.component.tsx:35]: Do we know about any use cases when `Date` is provided as a label ?

**Review by maciejrybaniec — CHANGES_REQUESTED** [review]

File: packages/charts/src/charts/donut/donut-chart.component.tsx (hunk 1 above)
Comment (dariuszlacheta) [reply packages/charts/src/charts/donut/donut-chart.component.tsx]: Corrected

File: packages/charts/src/charts/pie/pie-slice.component.tsx (hunk 2 above)
Comment (dariuszlacheta) [reply packages/charts/src/charts/pie/pie-slice.component.tsx:35]: Date can come from `formatValue` function

**Review by maciejrybaniec — APPROVED** [review]

**Pull request merged**